tests/units/test.h

```diff
@@ -1,6 +1,6 @@
 //#
-//# Copyright (C) 2020-2021 QuasarApp.
-//# Distributed under the lgplv3 software license, see the accompanying
+//# Copyright (C) 2021-2021 QuasarApp.
```

Comment:
Please restore time range of the copyright file. 
```suggestion
//# Copyright (C) 2020-2021 QuasarApp.
```